test/sanitize-field-names/_shared.js

```diff
@@ -96,6 +96,7 @@ function createAgentConfig(values = {}) {
     metricsInterval: 0,
     centralConfig: false,
     captureBody: 'all',
+    apmServerVersion: '8.17.0',
```

Comment:
Reviewer note: This removes a spurious log.error from the APM agent used in the test/sanitize-field-names/*.test.js tests. The APM agent is started without an APM server having been created. The APM agent then complains when trying to get the APM server's version.  We use this `apmServerVersion` technique in many other tests.